assignments/objects.js

```diff
@@ -49,16 +93,42 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: "Susan",
+  age: 70,
+  speak: function () {
+    return `My name is ${this.name}!`;
+  },
+  child: {
+    name: "George",
+    age: 50,
+    speak: function () {
+      return `My name is ${this.name}!`;
+    },
+    child: {
+      name: "Sam",
+      speak: function () {
+        return `My name is ${this.name}!`;
+      },
+      age: 30
+    } 
+  }
+}
 
 // Log the parent object's name
+console.log(parent.name);
 
 // Log the child's age
+console.log(parent.child.age);
 
 // Log the name and age of the grandchild
+console.log(parent.child.child.name + ", " + parent.child.child.age);
```

Comment:
this could also be written as a template literal using backticks and interpolated strings.
```console.log(`${parent.child.child.name}, ${parent.child.child.age}`);```